patch.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/src/Plugin/facets/widget/DateWidget.php b/src/Plugin/facets/widget/DateWidget.php
```

Comment:
?

Reply:
No clue how this got here, I'll remove it.